AlpacaHTTP/web/style.css

```
@@ -681,47 +681,142 @@ h2 {
+    color: #f1f1f1;
+    font-size: 0.95em;
+    word-break: break-all;
+}
+
+.log-file-viewer-body {
+    margin: 0;
+    padding: 12px;
+    max-height: 460px;
+    overflow: auto;
+    white-space: pre-wrap;
+    word-break: break-word;
```

Stylelint (17.11.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)